src/implementations/vanilla/components/page-header/page-header-print.scss

```diff
@@ -47,3 +47,9 @@
   margin-bottom: 0;
   margin-top: map.get(theme.$spacing-print, 'm');
 }
+
+.ecl-page-header--image {
```

Comment:
but wasn't the possibility to have these images in print one of the reasons to adopt the picture instead of a background image..?